istio-operator-1.19.advisories.yaml

```diff
@@ -0,0 +1,21 @@
+schema-version: "2"
+
+package:
+  name: istio-operator-1.19
+
+advisories:
+  - id: CVE-2019-3826
+    events:
+      - timestamp: 2023-09-25T20:27:23Z
+        type: false-positive-determination
+        data:
+          type: vulnerability-record-analysis-contested
```

Comment:
This should be this type: `vulnerable-code-version-not-used` - basically, we're saying it's the right component, but the vulnerable version isn't used in our package!

Reply:
Done! Thanks